### PR TITLE
Rancher 463

### PR DESCRIPTION
**Pull request by OHaimanov**

### CODE REVIEW

File: pipelines/rancher/deploy/ui-bundle-deploy.groovy [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    domains: [ui   : common.generateDomain(params.rancher_cluster_name, params.rancher_project_name, tenant.getId(), Constants.CI_ROOT_DOMAIN),
+              okapi: common.generateDomain(params.rancher_cluster_name, params.rancher_project_name, 'okapi', Constants.CI_ROOT_DOMAIN),
+              edge : common.generateDomain(params.rancher_cluster_name, params.rancher_project_name, 'edge', Constants.CI_ROOT_DOMAIN)],
+)
+
+project_model.uiBundleTag = params.ui_bundle_build ? "${project_model.getClusterName()}-${project_model.getProjectName()}-${tenant.getId()}-${project_model.getHash().take(7)}" : params.ui_bundle_tag
+
+ansiColor("xterm") {
+    common.refreshBuidParameters(params.refresh_parameters)
+    node("jenkins-agent-java11") {
```
Comment (kdubniak) [inline pipelines/rancher/deploy/ui-bundle-deploy.groovy:38]: Can we use here dynamic agents jobsParameters.agents(), ?

File: pipelines/rancher/deploy/ui-bundle-deploy.groovy [hunk 2]
```diff
@@ -0,0 +1,79 @@
+import org.folio.Constants
+import org.folio.rest.model.OkapiTenant
+import org.folio.utilities.model.Project
+
+@Library('pipelines-shared-library@RANCHER-463') _
```
Comment (kdubniak) [inline pipelines/rancher/deploy/ui-bundle-deploy.groovy:5]: ))

File: pipelines/rancher/deploy/ui-bundle-deploy.groovy [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+            stage("Checkout") {
+                checkout scm
+            }
+            stage("Read configs") {
+                project_model.modulesConfig = readYaml file: "${Constants.HELM_MODULES_CONFIG_PATH}/${project_model.getConfigType()}.yaml"
+            }
+
+            if (params.ui_bundle_build) {
+                stage("Build UI bundle") {
+                    build job: 'Rancher/UI-Build-463',
```
Comment (kdubniak) [inline pipelines/rancher/deploy/ui-bundle-deploy.groovy:49]: Rancher/UI-Build ?

File: pipelines/rancher/ui-build.groovy [hunk 4]
```diff
@@ -12,39 +15,48 @@ properties([
...
+String okapi_url = params.custom_url?.trim() ? params.custom_url : "https://" + project_model.getDomains().okapi
 
 ansiColor('xterm') {
-    if (params.refreshParameters) {
-        currentBuild.result = 'ABORTED'
-        error('DRY RUN BUILD, NO STAGE IS ACTIVE!')
-    }
-    node(params.agent) {
+    common.refreshBuidParameters(params.refresh_parameters)
+    node("jenkins-agent-java11") {
```
Comment (kdubniak) [inline pipelines/rancher/ui-build.groovy:46]: Also possible change to jobsParameters.agents() ?

File: pipelines/rancher/ui-build.groovy [hunk 5]
```diff
@@ -1,7 +1,10 @@
 #!groovy
 import org.folio.Constants
+import org.folio.rest.model.OkapiTenant
+import org.folio.utilities.model.Module
+import org.folio.utilities.model.Project
 
-@Library('pipelines-shared-library') _
+@Library('pipelines-shared-library@RANCHER-463') _
```
Comment (kdubniak) [inline pipelines/rancher/ui-build.groovy:7]: just for reminder

File: pipelines/rancher/deploy/ui-bundle-deploy.groovy (hunk 1 above)
Comment (OHaimanov) [reply pipelines/rancher/deploy/ui-bundle-deploy.groovy:38]: Just to reduce parameters number for end users. Cause here it is not required to specify java version

File: pipelines/rancher/ui-build.groovy (hunk 4 above)
Comment (OHaimanov) [reply pipelines/rancher/ui-build.groovy:46]: Just to reduce parameters number for end users. Cause here it is not required to specify java version

**Review by Volodymyr-Kartsev — APPROVED** [review]

**Review by vkapylou — APPROVED** [review]

**Pull request merged**